CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * Support dynamically changeable logging configuration of Kafka Connect and Kafka Connect S2I
 * Support dynamically changeable logging configuration of Kafka brokers
 * Support dynamically changeable logging configuration of Kafka MirrorMaker2
+* Updated Jaeger tracing libraries to support the W3C trace context headers
```

Comment:
I don't know what it is :-) ... but how much is it relevant to the user to know that the new Jaeger lib has this feature related to Kafka?

Reply:
Probably not really, there was a to-do box for updating the changelog so I added it :). But I agree that it's not really relevant to the end user. I'll remove it!